packaging/templates/template.yml

```diff
@@ -0,0 +1,7 @@
+name: 'Release {target}'
```

Comment:
Also these can be unified by taking a input: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-a-docker-container-action#create-an-action-metadata-file

Reply:
done.